frontend/src/components/shared_components/SpecialButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { string } from 'prop-types';

```

Comment:
We are not using prop-types, they shouldn't even be in the dependency tree. 
Did you add them? 
If yes, remove them please.

Reply:
I didn't add them, but I'm happy to remove 😁